testfiles/5ZoneCoolingPanelBaseboard.idf

```diff
@@ -2205,48 +2205,56 @@
-    0.3,                     !- Fraction Radiant
-    0.3,                     !- Fraction of Radiant Energy Incident on People
-    RIGHT-1,                 !- Surface 1 Name
-    0.4,                     !- Fraction of Radiant Energy to Surface 1
-    C2-1,                    !- Surface 2 Name
-    0.2,                     !- Fraction of Radiant Energy to Surface 2
-    SB25,                    !- Surface 3 Name
-    0.1;                     !- Fraction of Radiant Energy to Surface 3
+    SPACE2-1 Baseboard, !- Name
+    SPACE2-1 Baseboard Design, !- Design_Object_Name
+    ReheatCoilAvailSched, !- Availability_Schedule_Name
+    SPACE2-1 Zone Coil Water In Node, !- Inlet_Node_Name
+    SPACE2-1 Zone Coil Water Out Node, !- Outlet_Node_Name
+    87.78, !- Rated_Average_Water_Temperature
+    0.063, !- Rated_Water_Mass_Flow_Rate
+    autosize, !- Heating_Design_Capacity
+    autosize, !- Maximum_Water_Flow_Rate
+    RIGHT-1, !- Surface_1_Name
+    0.4, !- Fraction_of_Radiant_Energy_to_Surface_1
+    C2-1, !- Surface_2_Name
+    0.2, !- Fraction_of_Radiant_Energy_to_Surface_2
+    SB25, !- Surface_3_Name
+    0.1; !- Fraction_of_Radiant_Energy_to_Surface_3
+
+  ZoneHVAC:Baseboard:RadiantConvective:Water:Design,
+    SPACE2-1 Baseboard Design, !- Design_Object_Name
+    HeatingDesignCapacity, !- Heating_Design_Capacity_Method
+    , !- Heating_Design_Capacity_Per_Floor_Area
+    , !- Fraction_of_Autosized_Heating_Design_Capacity
+    0.001, !- Convergence_Tolerance
+    0.3, !- Fraction_Radiant
+    0.3; !- Fraction_of_Radiant_Energy_Incident_on_People
 
   ZoneHVAC:Baseboard:RadiantConvective:Water,
-    SPACE4-1 Baseboard,      !- Name
-    ReheatCoilAvailSched,    !- Availability Schedule Name
-    SPACE4-1 Zone Coil Water In Node,  !- Inlet Node Name
-    SPACE4-1 Zone Coil Water Out Node,  !- Outlet Node Name
-    82.22,                   !- Rated Average Water Temperature {C}
-    0.063,                   !- Rated Water Mass Flow Rate {kg/s}
-    HeatingDesignCapacity,   !- Heating Design Capacity Method
-    autosize,                !- Heating Design Capacity {W}
-    ,                        !- Heating Design Capacity Per Floor Area {W/m2}
-    ,                        !- Fraction of Autosized Heating Design Capacity
-    autosize,                !- Maximum Water Flow Rate {m3/s}
-    0.001,                   !- Convergence Tolerance
-    0.3,                     !- Fraction Radiant
-    0.3,                     !- Fraction of Radiant Energy Incident on People
-    LEFT-1,                  !- Surface 1 Name
-    0.4,                     !- Fraction of Radiant Energy to Surface 1
-    C4-1,                    !- Surface 2 Name
-    0.2,                     !- Fraction of Radiant Energy to Surface 2
-    SB45,                    !- Surface 3 Name
-    0.1;                     !- Fraction of Radiant Energy to Surface 3
+    SPACE4-1 Baseboard, !- Name
+    SPACE4-1 Baseboard Design, !- Design_Object_Name
```

Comment:
`SPACE4-1 Baseboard Design` is the same as `SPACE2-1 Baseboard Design`, I think we can use one design object for both baseboards. I think this is the main point of this new feature.

Reply:
@dareumnam  Thanks so much for the review! Will take off the redundant design objects.